lib/gui/lvgl/lvgl_display_mono.c

```diff
@@ -24,10 +24,6 @@ void lvgl_flush_cb_mono(struct _disp_drv_t *disp_drv,
 	desc.pitch = w;
 	desc.height = h;
 	display_write(display_dev, area->x1, area->y1, &desc, (void *) color_p);
```

Comment:
Note that `SCREEN_INFO_DOUBLE_BUFFER` in `screen_info` indicates that the display controller has two internal frame buffers, as `LVGL` only updates partial buffer space both buffers need to be updated else they are no longer in sync and the display will only show a partial result on the screen.

Reply:
I'm pretty sure that the ssd16xx display I'm using does not have two internal frame buffers. Each `display_write` call performs a "master activation" on the display, which is what causes the display to get updated from internal RAM. But it points me in the right direction for finding a proper fix, thank you very much.